releng/tools/org.openjdk.jmc.util.releasenotes/notes/9.0/notes.xml

```diff
@@ -0,0 +1,261 @@
+				for more information.
+			</description>
+			<image>general/issues.png</image>
+		</note>
+        <note version="9.0">
+            <title>Add user configuration for local JVM refresh interval</title>
+            <description>
+				Previously the JVM Browser checked every 5000 ms [0] for new JVMs. This can now be configured.
+            </description>
+            <image>general/refreshinterval.png</image>
+        </note>
+    </component>
+    <component>
+        <name>Core</name>
+        <!-- Core library enhancements for JMC 9 -->
+        <note version="9.0">
+            <title>Better JFR parser performance</title>
+            <description>
+            	Multiple efforts have been made to reduce allocations in the JMC parser, including:
+            	reduced allocation of Doubles, reduced allocation rate in ParserStats.
+            	Also, when duration events aren't ordered by their end time (e.g. events which stack so that the last 
+            	event finishes first, or file reads with overlaps) `DisjointBuilder.add` can be slow because of 
+            	the linear search for the lane, and then a linear time reordering. This has been improved with 
+            	a binary search.
+            </description>
+            <image>jfr/jfrperformance.png</image>
+        </note>        
+        <note version="9.0">
+            <title>Support checkpoint event sizes beyond u4 limit</title>
+            <description>
+            	The JMC JFR parser now support checkpoint event sizes beyond the u4 limit.
+            </description>
+            <image>jfr/checkpointsize.png</image>
+        </note>        
+        <note version="9.0">
+            <title>Move non-Eclipse dependant classes from org.openjdk.jmc.ui.common to org.openjdk.jmc.common</title>
+            <description>
+            	There are a number of classes currently in jmc.ui.common that would be a great asset to the core distribution (and the 
+            	third-party applications that consume jmc-core), and these classes now live in jmc.common. Please see JMC-7308 
+            	for further information.
+            </description>
+            <image>general/reorganize.png</image>
+        </note>
+        <note version="9.0">
+            <title>Move rjmx bundle from application to core</title>
+            <description>
+				The rjmx classes and related services (FlightRecorderService) are now exposed for third-party application usage.
+				Please see JMC-7069 for further information.
+            </description>
+            <image>general/reorganize.png</image>
+        </note>
+        <note version="9.0">
+            <title>Move org.openjdk.jmc.flightrecorder.configuration bundle from application to core</title>
+            <description>
+				The org.openjdk.jmc.flightrecorder.configuration bundle contains many classes useful for working with jfr, 
+				and would be a great asset to have in core. Please see JMC-7307 for further information.
```

Comment:
Maybe re-word the last part of that first sentence to something like:

s/and would be a great asset to have in core./and are now made available in core./